flask_mailgun.py

```diff
@@ -101,27 +179,33 @@ def send(self, **data):
         from_ = data.pop('from_', None)
         if from_ is None:
             from_ = 'no-reply@{}'.format(self.domain)
+
         data['from'] = from_
+
         if 'html' in data and 'text' not in data:
             data['text'] = html2text(data['html'])
+
         if self.debug:
             data.setdefault('html', '(no HTML)')
             current_app.logger.debug(debug_template, data)
             return
-        url = 'https://api.mailgun.net/v3/{}/messages'.format(self.domain)
+
+        url = self.api_url_template.format(domain=self.domain)
+
         response = requests.post(url, auth=('api', self.key), data=data)
         if response.status_code != 200:
-            raise APIError(response)
-        current_app.logger.info('Mailgun sent to %s', data['to'])
+            raise APIError(response, self.api_error_template)
 
+        current_app.logger.info(
```

Comment:
The logging module accepts the formatting string separate from the arguments for a reason. It formats the string only if it will be actually logged. I know it looks weird when the whole rest of the code uses different formatting method, but that's just how it is.